types/react-mentions-click/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "private": true,
+    "name": "@types/react-mentions-click",
+    "version": "1.0.9999",
+    "projects": [
+        "https://github.com/signavio/react-mentions"
```

Comment:
This should be `react-mention-clicks`'s repo

Reply:
What is the originating repo? I can find search hits for https://github.com/AkashLakade05/react-mentions-click but that's deleted.